projects/packages/videopress/src/client/block-editor/utils/video.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Pretty basic helper to check if a file is a video
+ * based on its mime type.
+ *
+ * @param {File} file - File to check.
+ * @returns {boolean}   Whether the file is a video.
+ */
+export function isVideoFile( file: File ): boolean {
+	if ( ! file?.type ) {
+		return false;
+	}
+
+	return file.type.startsWith( 'video/' );
```

Comment:
`Jetpack_Editor_Initial_State.allowedMimeTypes
```suggestion
	const allowedMimeTypes = Object.values(
		window?.Jetpack_Editor_Initial_State?.allowedMimeTypes ?? {}
	).filter( type => type.startsWith( 'video/' ) );
	return allowedMimeTypes.includes( file.type );
```

Reply:
To filter only the allowed mimetypes

Reply:
Nice suggestion. We need to ensure the right functionality also in the VideoPress standalone plugin.

Reply:
ref p1690387124731989/1690369223.294339-slack-C03TA48NSUX
For future reference, `Jetpack_Editor_Initial_State` is only available with Jetpack, not when using only VP.